packages/tables/examples/selectable-table.md

```diff
@@ -66,7 +66,7 @@ const isSelectAllChecked = (selectedRows, rows) => {
   <Body>
     {state.rows.map(row => (
       <Row key={row.id} selected={state.selectedRows[row.id]}>
```

Comment:
I noticed that the selected row styling was missing. I think renaming the `selected` prop to `isSelected` would fix it.

Notice: styling is not applied:
![TruncatedTitle](https://user-images.githubusercontent.com/1811365/70356963-82c71680-182a-11ea-9457-30118fd826bb.gif)